services/notification/src/helpers/database-helper.js

```diff
@@ -48,19 +48,6 @@ class DatabaseHelper {
                     MessageThreadStatus.PENDING,
                   ],
                 },
-                $or: [
-                  {
-                    lastAccess: null,
-                  },
-                  {
-                    lastAccess: {
-                      $lt: DateHelper.subtractMinutes(
-                        new Date(),
-                        this.instantUnreadLookbackInterval,
-                      ),
-                    },
-                  },
-                ],
```

Comment:
This logic will need to be replaced with something else, since it is necessary to help the lambda decide when to send email notifications, otherwise we will have to keep the lastAccess on the participant level, and just update all the threads, but I think the way you did it is better, we just need a solution for this one.
I think Manny might have a better idea.
Cc @mannykary @ddveloper

Reply:
I think we do lookup the user later in the same pipeline so that might be useful.